### PR TITLE
CI: collect golang unit test code coverage

### PR DESCRIPTION
**Pull request by nic-chen**
relate to #545

### CODE REVIEW

**Comment by membphis** [comment]
@nic-chen please update the description of this issue.

image

**Comment by nic-chen** [comment]
> @nic-chen please update the description of this issue.
> 
> image

updated.

I wanted to test it in my own repo, but accidentally submitted it here..

**Comment by juzhiyuan** [comment]
Emmm Not sure if this PR is going to review, kindly mark it as draft please.

File: Makefile [hunk 1]
```diff
@@ -15,11 +15,18 @@
...
 ifeq ("$(wildcard .actions/openwhisk-utilities/scancode/scanCode.py)", "")
 	git clone https://github.com/apache/openwhisk-utilities.git .actions/openwhisk-utilities
 	cp .actions/ASF* .actions/openwhisk-utilities/scancode/
 endif
-	.actions/openwhisk-utilities/scancode/scanCode.py --config .actions/ASF-Release.cfg ./
\ No newline at end of file
+	.actions/openwhisk-utilities/scancode/scanCode.py --config .actions/ASF-Release.cfg ./
+
+
+.PHONY: api-test
```
Comment (membphis) [inline Makefile:30]: please add a comment like this line: https://github.com/apache/apisix-dashboard/blob/90c11e50385b3b9dbde49c0b1c6e0f31c00995e0/Makefile#L20

then we can get the command list: `make help`

**Comment by membphis** [comment]
@nic-chen I think we can merge this PR soon. change the title to `Collect golang unit test code coverage`

**Comment by membphis** [comment]
@nic-chen please resolve the conflicting files.

File: .github/workflows/test-api.yml [hunk 2]
```diff
@@ -50,6 +50,7 @@ jobs:
         run: |
           bash <(curl -s https://codecov.io/bash)
 
+
```
Comment (membphis) [inline .github/workflows/test-api.yml:53]: one blank line is enough

**Review by membphis — APPROVED** [review]

**Pull request merged**